apps/sample-app/src/components/Calendar.tsx

```diff
@@ -0,0 +1,21 @@
+import './styles.css';
+
+import { Button } from '@fluentui/react-components';
+import { calendar } from '@microsoft/teams-js';
+import React from 'react';
+
+export const Calendar: React.FunctionComponent = () => {
+  const handleCalendar = async (): Promise<void> => {
+    // create calendar paramaters
+    const calendarParams: calendar.ComposeMeetingParams = {
+      attendees: ['t-abarthakur@microsoft.com'],
```

Comment:
You may want to replace this with a placeholder value instead of using your real address.

Reply:
Something like 'emailaddress@company.com'